### PR TITLE
Made sure empty cells don't get logged therefore preventing 400 error's

### LINKED ISSUE #180: Bug: 400 status code when hovering over empty line number
When hovering over the left-hand column of a unified diff (see image below), the database gives a 400 Bad Request because the line number is not available.
This issue should be fixed by discarding the event if this is the case.

### PR DESCRIPTION
**Pull request by Exclaminator**
Will close #180.

Made sure empty cells don't get logged therefore preventing 400 error's

### CODE REVIEW

**Comment by mpsijm** [comment]
I am now getting 400 errors when hovering over lines of code?

File: src/octopeer-github/content/ElementSelectionBehaviour/ElementSelectionBehaviourData.ts [hunk 1]
```diff
@@ -257,6 +257,6 @@ const elementSelectionBehaviourDataList: ElementSelectionBehaviourData[] = [
             const line_number = parseInt($(eventObject.currentTarget).children(".add-line-comment").attr("data-line"), 10);
             return EventFactory.semantic(ElementID.DIFF_LINE_OF_CODE, eventID, commit_hash, filename, line_number);
         },
-        selector: ".blob-code-addition, .blob-code-deletion, .blob-code-context",
+        selector: ".blob-code-addition:not(.empty-cell), .blob-code-deletion:not(.empty-cell), .blob-code-context:not(.empty-cell)",
```
Comment (mpsijm) [inline src/octopeer-github/content/ElementSelectionBehaviour/ElementSelectionBehaviourData.ts]: Does a line of code ever have the `.empty-cell` class?

**Comment by Exclaminator** [comment]
@mpsijm and me tried replicating his error on my machine, seems to be working properly here.

**Comment by Exclaminator** [comment]
@aaronang will change this in the database.

**Comment by mdingjan** [comment]
I couldn't break it; I like it. :)
Add some more tests and I'ma happy duck. :)
After that, ready for merging.

**Comment by mpsijm** [comment]
Okay, I've added some more tests and it all works now. A few more reviews and this can be merged :)

**Comment by mdingjan** [comment]
Thanks for adding the extra tests! Ready for merge :)

**Comment by thervh70** [comment]
+0.07 i like this :p 

Looking good and unbreakable, will merge!

**Pull request merged**